modules/cluster/variables.tf

```diff
@@ -36,12 +36,13 @@ variable "instance_key" {
 
 variable "instance_volume_size" {
   description = "The size of the volume in gigabytes."
-  default     = "8"
+  default     = "30"
 }
 
-variable "docker_volume_size" {
-  description = "The size of the docker volume (/dev/xvdcz) in gigabytes."
-  default     = "22"
+variable "ebs_block_devices" {
```

Comment:
Do we need to add this variable to the spotfleet cluster also?

Reply:
The spotfleet module has not supported variable volume sizes or additional volumes before - will write a ticket to add it as a new feature